docs/examples/plotting_functions/heatmap.md

```diff
@@ -82,6 +82,61 @@ heatmap(xs, ys, zs)
 ```
 \end{examplefigure}
 
-### Colors
+### Colorbar for single heatmap
 
-Using the previous example, we can see how to change the color of our plot. You can find additional colors [here](https://makie.juliaplots.org/stable/documentation/colors/index.html#colors).
+To get a scale for what the colors represent, add a colorbar. The colorbar is 
+placed within the figure in the first argument, and the scale and colormap can be 
+conveniently set by passing the relevant heatmap to it.
+
+\begin{examplefigure}{}
+```julia
+using CairoMakie
+CairoMakie.activate!() # hide
+Makie.inline!(true) # hide
+
+xs = range(0, 2π, length=100)
+ys = range(0, 2π, length=100)
+zs = [sin(x*y) for x in xs, y in ys]
+
+fig, ax, hm = heatmap(xs, ys, zs)
+Colorbar(fig[:, end+1], hm)
+
+fig
+```
+\end{examplefigure}
+
+### Colorbar for multiple heatmaps
+
+When there are several heatmaps in a single figure, it can be useful
+to have a single colorbar represent all of them. It is important to then 
+have syncronized scales and colormaps for the heatmaps and colorbar. This is done by
+setting the colorrange explicitly, so that it is independent of the data shown by 
+that particular heatmap.
+
+Since the heatmaps in the example below have the same colorrange and colormap, any of them 
+can be passed to `Colorbar` to give the colorbar the same attributes. Alternativly, 
+the colorbar attributes can be set explicitly.
+
+\begin{examplefigure}{}
+```julia
+using CairoMakie
+CairoMakie.activate!() # hide
+Makie.inline!(true) # hide
+
+xs = range(0, 2π, length=100)
+ys = range(0, 2π, length=100)
+zs1 = [sin(x*y) for x in xs, y in ys]
+zs2 = [2sin(x*y) for x in xs, y in ys]
+
+my_limits = extrema(hcat(zs1, zs2))  # Get maximum and minimum for all z-values
```

Comment:
I don't like this too much because I can see the `hcat` breaking on different datasets already, where the matrices don't match. I haven't been able to come up with a really nice and short "cumulative extrema of multiple arrays" expression though that's easy to understand. Maybe it's fine just specifying `-2, 2` here manually as we know that from the `2sin` expression.

Reply:
How about `extrema(extrema.((zs1, zs2)))`

Reply:
That makes a tuple of tuples

Reply:
Ah yeah...
@KronosTheLate, should we just use `(-2, 2)`... I think that's fine here, since we're not trying to show how to get the extrema from the values ;)

Reply:
I think that's fair, it's reasonably obvious with the sine (actually it can even look better because if the limits go from -1.99123 to 1.992852 then the colorbar ticks don't have -2 and 2)

Reply:
Ok, I'll edited it... @KronosTheLate I think we can merge it now, if you don't have objections

Reply:
No objections - extrema of extrema is besides the point of this documentation, so manual limits, though inelegant in some ways, seems like a great solution for this documentation ^_^